api/users/serializers.py

```diff
@@ -19,7 +19,7 @@ class QuickFilesRelationshipField(RelationshipField):
 
     def to_representation(self, value):
         relationship_links = super(QuickFilesRelationshipField, self).to_representation(value)
-        quickfiles_guid = QuickFilesNode.objects.get_for_user(value)._id
+        quickfiles_guid = QuickFilesNode.objects.filter(creator=value).values_list('guids___id').get()
```

Comment:
You can use the reverse relation here, which will take advantage of Django's query caching:

```python
value.created.filter(type='osf.quickfilesnode').values_list('guids___id').get()
```